#168393052 fix model for handling user/manager comment relating to trip request made

torsami77 opened the pull request:
#### Description of Task to be completed?
- Fix the model to handling CRUD operation for user/manager comment relating to trip request

 #### How should this be manually tested?
(On Existing database)
- Drop the Comments table
- run ```npm run undo-migrate ``` from Terminal or Delete 'create-7-comments.js' from the 'SequelizeMeta' table in your database
- Run ```npm run migrate```
- Run ```npm run test```

 #### Any background context you want to provide?
- Developers working with this model should create controllers that reflects the task in their PT story retrospectively.

 #### What are the relevant pivotal tracker stories?
[#168393052](https://www.pivotaltracker.com/story/show/168393052)

 #### Screenshots (if appropriate)
 #### Questions:

sa-ma commented:
@torsami77 I think all we have to do is `npm run undo-migrate` before running migrations instead of deleting them individually

sa-ma commented:
You have to edit your commit it shows @funmi5 as the author.

oyewoas commented on src/models/Comment.js:
```diff
@@ -3,7 +3,30 @@ export default (sequelize, DataTypes) => {
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'SET NULL'
     }
```
well done @torsami77 I think you don't need to repeat the requestId and userId here again since  you already referenced them in the `database/migrations/create-7-comments.js file

oyewoas commented on src/test/models/Comment.test.js:
```diff
@@ -14,6 +15,18 @@ describe('Test for Comment Model', () => {
 
   context('properties', () => {
-    ['comment'].forEach(checkPropertyExists(comment));
+    ['comment', 'userId', 'requestId'].forEach(checkPropertyExists(comment));
```
nice @torsami77 , i think you will have to just use `comment` here for the test to finally pass

torsami77 replied:
Thanks team, have made the changes

torsami77 committed:
fix(model for comments): fix model for handling user comment relating to trip request made

- fix model for user or manager to be able to post a comment relating to a request made

[Delivers #168393052]
